examples/samplers/SMC2_gaussians.ipynb

```diff
@@ -7,7 +7,7 @@
     "# Sequential Monte Carlo\n",
```

Comment:
<p>testval should be replaced by initval as the warning suggests </p><p> </p><p>Seems like in sample SMC we should suppress the Potential warning wheb sampling from the prior to start the population...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/334/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>testval replaced.</p><p> </p><p>I tried to filter the warning, but then the progressbar does not progress :-(</p><p> </p>

Reply:
> I tried to filter the warning, but then the progressbar does not progress :-

I mean internally in PyMC

Reply:
I tried to catch the warning inside the initialize_population function.

Reply:
How did you do it?

Reply:
see this https://github.com/pymc-devs/pymc/pull/5827